diff -Nur gaim-audacious-remote-0.1/src/gaim-audacious-remote.c gaim-audacious-remote-0.1_patched/src/gaim-audacious-remote.c
--- gaim-audacious-remote-0.1/src/gaim-audacious-remote.c	2006-03-10 07:20:16.000000000 +0200
+++ gaim-audacious-remote-0.1_patched/src/gaim-audacious-remote.c	2006-11-23 19:48:39.000000000 +0200
@@ -24,11 +24,12 @@
 
 #include <audacious/beepctrl.h>
 
+#include <string.h>
 #ifdef HAVE_CONFIG_H
 # include "../gxr_config.h"
 #endif
 
-#include <internal.h>
+#define GAIM_PLUGINS
 
 #include <cmds.h>
 #include <conversation.h>
@@ -1191,12 +1192,12 @@
 
 static const GtkStockItem stock_items[] =
 {
-	{ GXR_STOCK_XMMS,     N_(""),     0, 0, NULL },
-	{ GXR_STOCK_PREVIOUS,     N_(""),     0, 0, NULL },
-	{ GXR_STOCK_PLAY,     N_(""),     0, 0, NULL },
-	{ GXR_STOCK_PAUSE,     N_(""),     0, 0, NULL },
-	{ GXR_STOCK_STOP,     N_(""),     0, 0, NULL },
-	{ GXR_STOCK_NEXT,     N_(""),     0, 0, NULL }
+	{ GXR_STOCK_XMMS,     "",     0, 0, NULL },
+	{ GXR_STOCK_PREVIOUS, "",     0, 0, NULL },
+	{ GXR_STOCK_PLAY,     "",     0, 0, NULL },
+	{ GXR_STOCK_PAUSE,    "",     0, 0, NULL },
+	{ GXR_STOCK_STOP,     "",     0, 0, NULL },
+	{ GXR_STOCK_NEXT,     "",     0, 0, NULL }
 };
 
 static void
