--- file-roller-2.14.1/src/fr-command-rar.c	2006-04-03 12:47:17.000000000 +0300
+++ fr-command-rar.c	2006-06-02 07:22:05.000000000 +0300
@@ -250,7 +250,7 @@
 	add_password_arg (comm, password, FALSE);
 
 	/* disable percentage indicator */
-	fr_process_add_arg (comm->process, "-idp");
+	fr_process_add_arg (comm->process, "-Idp");
 	
 	/* stop switches scanning */
 	fr_process_add_arg (comm->process, "--"); 
@@ -316,7 +316,7 @@
 	add_password_arg (comm, password, TRUE);
 
 	/* disable percentage indicator */
-	fr_process_add_arg (comm->process, "-idp");
+	fr_process_add_arg (comm->process, "-Idp");
 	
 	/* stop switches scanning */
 	fr_process_add_arg (comm->process, "--"); 
@@ -350,7 +350,7 @@
 	add_password_arg (comm, password, TRUE);
 
 	/* disable percentage indicator */
-	fr_process_add_arg (comm->process, "-idp");
+	fr_process_add_arg (comm->process, "-Idp");
 
 	/* stop switches scanning */
 	fr_process_add_arg (comm->process, "--");
