Index: src/com/googlecode/jkota/swing/SwingSettings.java
===================================================================
--- src/com/googlecode/jkota/swing/SwingSettings.java	(revision 76)
+++ src/com/googlecode/jkota/swing/SwingSettings.java	(revision 77)
@@ -66,6 +66,7 @@
 		setResizable(false);
 		SwingUtil.center(this);
 		setDefaultCloseOperation(DO_NOTHING_ON_CLOSE);
+		hideShowAPIKey();
 		setVisible(true);
 	}
 
@@ -83,15 +84,18 @@
 			}
 			dispose();
 		}
-		else if("list".equals(e.getActionCommand())) {
-			if(updaterList.getSelectedItem().equals("TTNet ADSL")) {
-				apiKeyLabel.setVisible(true);
-				apiKey.setVisible(true);
-			}
-			else {
-				apiKeyLabel.setVisible(false);
-				apiKey.setVisible(false);
-			}
+		else if("list".equals(e.getActionCommand()))
+			hideShowAPIKey();
+	}
+	
+	private void hideShowAPIKey() {
+		if(updaterList.getSelectedItem().equals("TTNet ADSL")) {
+			apiKeyLabel.setVisible(true);
+			apiKey.setVisible(true);
 		}
-	}	
+		else {
+			apiKeyLabel.setVisible(false);
+			apiKey.setVisible(false);
+		}
+	}
 }
