diff --git a/imdb/imdb-get.sh b/imdb/imdb-get.sh
index ab93b77..72e4799 100644
--- a/imdb/imdb-get.sh
+++ b/imdb/imdb-get.sh
@@ -130,7 +130,7 @@ doIMDB() {
        fi
    fi
 
-   if [[ "$UPDATEFILENAME" == "true" || "$REALREFRESH" == "true" ]]; then
+   if [[ ( "$UPDATEFILENAME" == "true" || "$REALREFRESH" == "true") && "$FILENAME" != "" ]]; then
       # update index
       NEWFILE="`realpath "$FILENAME"`"
       if [[ "$INDEX" != ""  ]]; then
